### ISSUE TITLE
build_and_run-with-docker.sh not works (develop branch)

### ISSUE THREAD

**Issue by adaminfinitum** [Asqatasun/Asqatasun#310]
## I'm trying to contribute but wget returns 'not found'

I'm running `Docker version 19.03.8, build afacb8b` on a current Mac.

I made a fork, cloned it, and `cd` into the project root, made sure I was on the develop branch and:
` ./docker/build_and_run-with-docker.sh --source-dir $(pwd) --docker-dir docker/SNAPSHOT-local`

returns

 ```
  Help documentation for build_and_run-with-docker.sh
  --------------------------------------------------------------
  This script launches a sequence that:
    - builds Asqatasun from sources
    - builds a new Docker image
    - runs   a new Docker container
    - execute functional tests    (optional)
    - execute krash test campaign (optional)
```

So, I decided maybe the `$(pwd)` was the problem, OK:
` ./docker/build_and_run-with-docker.sh --source-dir /Users/a/github/Asqatasun --docker-dir docker/SNAPSHOT-local`

returns the same (help documentation)

So then I went to Docker Hub where I noticed that your Docker uses Ubuntu 14.04, but it is easy enough to install OpenJDK-8 on 18.04 so I decided to customize the `Dockerfile` found on hub.docker.com

`FROM ubuntu:18.04:latest`

Looking through the code I saw references to Postgres and Tomcat8.

So I decided to update it all so I could use Google Cloud Platform's CloudSQL, with Cloud Build and Cloud Run (everything has to be in a single `Dockerfile` that listens on :8080 but the Cloud SQL connection has to use a Unix Socket, not a port--should be easy enough).

Now to just customize the .conf files and make sure all my paths are right to connect it to a CloudSQL instance, build it on Cloud Build and Run it on Cloud Run, then submit a pull request once I have it all working...

## Observed behaviour

`wget https://github.com/Asqatasun/Asqatasun/releases/download/v5.0-SNAPSHOT/asqatasun-5.0-SNAPSHOT.i386.tar.gz`

```
--2020-03-20 01:49:15--  https://github.com/Asqatasun/Asqatasun/releases/download/v5.0-SNAPSHOT/asqatasun-5.0-SNAPSHOT.i386.tar.gz
  server: GitHub.com
  status: 404 Not Found
```
**Oops my mistake, looks like the 5.0-SNAPSHOT `Dockerfile` actually loads from:**

`wget http://binaries.asqatasun.org/asqatasun-5.0-SNAPSHOT.i386.tar.gz`

```
--2020-03-20 01:50:23--  http://binaries.asqatasun.org/asqatasun-5.0-SNAPSHOT.i386.tar.gz
Resolving binaries.asqatasun.org (binaries.asqatasun.org)... failed: nodename nor servname provided, or not known.
wget: unable to resolve host address ‘binaries.asqatasun.org’
```
**Darn it.**

## Expected behaviour
I was hoping to get a copy of the 5.0-SNAPSHOT on my local system to look over, modify and update. 

## Steps to reproduce the behaviour
See above

## Your configuration
Latest Docker (on Mac)
* Asqatasun version (in the footer of any webpage) -->
* Using Docker image ? -->
I copied the `Dockerfile` from hub.docker.com and started updating it directly to try to build an image from Ubuntu 18.04 instead of 14.04

It seems that's intentional, when I tried to
 ` wget https://github.com/Asqatasun/Asqatasun/releases/download/v4.1.0-rc.4/asqatasun-4.1.0-rc.4.i386.tar.gz`

```
  HTTP/1.1 302 Found
  expect-ct: max-age=2592000, report-uri="https://api.github.com/_private/browser/errors"
  ...
b341ae00-5877-11ea-9bd1-02cad7691dad?X-Amz- 100%[========================================================================================>]  78.73M  12.0MB/s    in 5.4s    
```
It downloaded something, but it was a `.tar.gz` file.

Anyway, I ran ` docker pull asqatasun/asqatasun:v4.1.0-rc.4` and have a local copy of the latest but I'll work on updating and send you a pull request if I can access the code base.

**Comment by jkowalczyk** [comment]
Hi Adam,

I've just tried to run the script. I get the error with the  `${pwd}` but it works as well with the plain path.
Are you sure your `pwd` command returns `/Users/a/github/Asqatasun`

The current  5.0-SNAPSHOT develop branch is not released yet. That's why you can't download any copy of it (wget on error). The only thing you can do, is build locally and try to run with the docker script.

I'm currently working on it to update, remove, simplify a couple of stuff. All is not ready yet :)

**Comment by dzc34** [comment]
I use this following command line and it works.
```bash
./docker/build_and_run-with-docker.sh --source-dir ${PWD} --docker-dir docker/SNAPSHOT-local
```
but I have some maven build errors:
- package javax.xml.bind.annotation does not exist
-  cannot find symbol class XmlTransient

```
[INFO] Reactor Summary:
[INFO] 
[INFO] asqatasun 5.0-SNAPSHOT ............................. SUCCESS [  1.231 s]
[INFO] engine 5.0-SNAPSHOT ................................ SUCCESS [  0.020 s]
[INFO] asqatasun-api 5.0-SNAPSHOT ......................... FAILURE [  1.190 s]
[INFO] rule-implementation-loader 5.0-SNAPSHOT ............ SKIPPED
(...)
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  3.010 s
[INFO] Finished at: 2020-03-25T01:28:37+01:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.1:compile (default-compile) on project asqatasun-api: Compilation failure: Compilation failure: 
[ERROR] engine/asqatasun-api/src/main/java/org/asqatasun/analyser/AnalyserService.java:[27,33] package javax.xml.bind.annotation does not exist
[ERROR] engine/asqatasun-api/src/main/java/org/asqatasun/analyser/AnalyserService.java:[33,2] cannot find symbol
[ERROR]   symbol: class XmlTransient

**Comment by dzc34** [comment]
@adaminfinitum, if you want to help and contribute, you can start by compiling with the following command:
```bash
cd <projectRoot>
mvn clean install
```

In case of a build problem, you can create an issue by specifying the commit you're building on.

If there are no errors, it will be time to look at how to make the docker image work.

**Comment by adaminfinitum** [comment]
Hi, sorry for the delayed response.  I recently installed  Maven a few days ago and have been learning how to use it so the clear directions are helpful.

I tried every imaginable combination of relative and absolute paths in the command:
`./docker/build_and_run-with-docker.sh --source-dir /Users/a/github/Asqatasun --docker-dir docker/SNAPSHOT-local`

All that I get is the `--help` page.

I also tried updating the shell script to use `#!/usr/local/bin/bash` instead of `#!/bin/bash` because I have a newer version of bash installed in my user space but that didn't help.

In the Maven documentation it explains that it may have errors the first few times you run it (so far I have run it 3 times to build HTMLUnit and no have finished successfully but the number of errors has decreased every time)...just now when I ran `mvn clean install` it failed...sorry about the log dump here I'm not sure what's important and what's not in this output (the ellipses  are `Downloading...` lines).

```
~/github/Asqatasun on develop*
$ mvn clean install
[INFO] Scanning for projects...
[WARNING] The project org.asqatasun:asqatasun:pom:5.0-SNAPSHOT uses prerequisites which is only intended for maven-plugin projects but not for non maven-plugin projects. For such purposes you should use the maven-enforcer-plugin. See https://maven.apache.org/enforcer/enforcer-rules/requireMavenVersion.html
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Build Order:
[INFO] 
[INFO] asqatasun                                                          [pom]
[INFO] engine                                                             [pom]
[INFO] asqatasun-api                                                      [jar]
[INFO] rule-implementation-loader                                         [jar]
[INFO] nomenclatureloader                                                 [jar]
[INFO] asqatasun-model                                                    [jar]
[INFO] contentloader                                                      [jar]
[INFO] contentadapter                                                     [jar]
[INFO] commons-processing                                                 [jar]
[INFO] processor                                                          [jar]
[INFO] consolidator                                                       [jar]
[INFO] asqatasun-services                                                 [jar]
[INFO] analyser                                                           [jar]
[INFO] logging                                                            [jar]
[INFO] scenarioloader                                                     [jar]
[INFO] asqatasun-engine                                                   [jar]
[INFO] asqatasun-resources                                                [jar]
[INFO] asqatasun-persistence                                              [jar]
[INFO] testing-tools                                                      [pom]
[INFO] rules-testing-tools                                                [jar]
[INFO] asqatasun-maven-kbcsv-plugin                              [maven-plugin]
[INFO] asqatasun-maven-testgenerator-plugin                      [maven-plugin]
[INFO] tgol-testing-tools                                                 [jar]
[INFO] rules-doc-utils                                                    [jar]
[INFO] rules                                                              [pom]
[INFO] rules-commons                                                      [jar]
[INFO] rules-creation-demo                                                [jar]
[INFO] rules-resources                                                    [jar]
[INFO] rules-rgaa3.0                                                      [jar]
[INFO] rules-rgaa3.2016                                                   [jar]
[INFO] rules-rgaa3.2017                                                   [jar]
[INFO] rules-accessiweb2.2                                                [jar]
[INFO] rules-seo1.0                                                       [jar]
[INFO] web-app                                                            [pom]
[INFO] email-sender                                                       [jar]
[INFO] security                                                           [jar]
[INFO] tgol-resources                                                     [jar]
[INFO] tgol-api                                                           [jar]
[INFO] tgol-si                                                            [jar]
[INFO] tgol-data-presentation                                             [jar]
[INFO] tgol-persistence                                                   [jar]
[INFO] tgol-orchestrator                                                  [jar]
[INFO] tgol-voter                                                         [jar]
[INFO] tgol-report                                                        [jar]
[INFO] tgol-security                                                      [jar]
[INFO] asqatasun-web-app                                                  [war]
[INFO] runner                                                             [pom]
[INFO] asqatasun-runner                                                   [jar]
[INFO] asqatasun-documentation                                            [jar]
[INFO] 
[INFO] ----------------------< org.asqatasun:asqatasun >-----------------------
[INFO] Building asqatasun 5.0-SNAPSHOT                                   [1/49]
[INFO] --------------------------------[ pom ]---------------------------------
Downloading from central: https://repo.maven.apache.org/maven2/org/codehaus/mojo/buildnumber-maven-plugin/1.4/buildnumber-maven-plugin-1.4.pom
(...)
Downloaded from central: https://repo.maven.apache.org/maven2/org/codehaus/plexus/plexus-utils/3.0.21/plexus-utils-3.0.21.jar (245 kB at 289 kB/s)
[INFO] ShortRevision tag detected. The value is '7'.
[INFO] Executing: /bin/sh -c cd '/Users/a/github/Asqatasun' && 'git' 'rev-parse' '--verify' '--short=7' 'HEAD'
[INFO] Working directory: /Users/a/github/Asqatasun
[INFO] Storing buildNumber: 2795adf at timestamp: 1585118763793
[INFO] Storing buildScmBranch: develop
[INFO] 
[INFO] --- jacoco-maven-plugin:0.8.5:prepare-agent (default) @ asqatasun ---
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/maven/shared/file-management/1.2.1/file-management-1.2.1.pom
(...)
Downloaded from central: https://repo.maven.apache.org/maven2/com/vladsch/flexmark/flexmark-ext-wikilink/0.27.0/flexmark-ext-wikilink-0.27.0.jar (25 kB at 45 kB/s)
[INFO] Pre-processing markdown files from input directory: /Users/a/github/Asqatasun
[INFO] Process Pegdown extension options
[INFO] Pegdown extension TABLES
[INFO] Pegdown extension FENCED_CODE_BLOCKS
[INFO] Pegdown extension options = 160
[INFO] Parse Markdown to HTML
[INFO] 
[INFO] --- jacoco-maven-plugin:0.8.5:report (report) @ asqatasun ---
[INFO] Skipping JaCoCo execution due to missing execution data file.
[INFO] 
[INFO] --- maven-install-plugin:2.4:install (default-install) @ asqatasun ---
Downloading from central: https://repo.maven.apache.org/maven2/org/codehaus/plexus/plexus-utils/3.0.5/plexus-utils-3.0.5.pom
(...)
Downloaded from central: https://repo.maven.apache.org/maven2/org/codehaus/plexus/plexus-utils/3.0.5/plexus-utils-3.0.5.jar (230 kB at 2.5 MB/s)
[INFO] Installing /Users/a/github/Asqatasun/pom.xml to /Users/a/.m2/repository/org/asqatasun/asqatasun/5.0-SNAPSHOT/asqatasun-5.0-SNAPSHOT.pom
[INFO] 
[INFO] ------------------------< org.asqatasun:engine >------------------------
[INFO] Building engine 5.0-SNAPSHOT                                      [2/49]
[INFO] --------------------------------[ pom ]---------------------------------
[INFO] 
[INFO] --- maven-clean-plugin:3.0.0:clean (default-clean) @ engine ---
[INFO] 
[INFO] --- build-helper-maven-plugin:3.0.0:timestamp-property (timestamp-property) @ engine ---
[INFO] 
[INFO] --- buildnumber-maven-plugin:1.4:create (default) @ engine ---
[INFO] ShortRevision tag detected. The value is '7'.
[INFO] Executing: /bin/sh -c cd '/Users/a/github/Asqatasun/engine' && 'git' 'rev-parse' '--verify' '--short=7' 'HEAD'
[INFO] Working directory: /Users/a/github/Asqatasun/engine
[INFO] Storing buildNumber: 2795adf at timestamp: 1585118772661
[INFO] Storing buildScmBranch: develop
[INFO] 
[INFO] --- jacoco-maven-plugin:0.8.5:prepare-agent (default) @ engine ---
[INFO] argLine set to -javaagent:/Users/a/.m2/repository/org/jacoco/org.jacoco.agent/0.8.5/org.jacoco.agent-0.8.5-runtime.jar=destfile=/Users/a/github/Asqatasun/engine/target/jacoco.exec
[INFO] 
[INFO] --- markdown-page-generator-plugin:2.1.0:generate (default) @ engine ---
[INFO] Pre-processing markdown files from input directory: /Users/a/github/Asqatasun/engine
[INFO] 
[INFO] --- jacoco-maven-plugin:0.8.5:report (report) @ engine ---
[INFO] Skipping JaCoCo execution due to missing execution data file.
[INFO] 
[INFO] --- maven-install-plugin:2.4:install (default-install) @ engine ---
[INFO] Installing /Users/a/github/Asqatasun/engine/pom.xml to /Users/a/.m2/repository/org/asqatasun/engine/5.0-SNAPSHOT/engine-5.0-SNAPSHOT.pom
[INFO] 
[INFO] --------------------< org.asqatasun:asqatasun-api >---------------------
[INFO] Building asqatasun-api 5.0-SNAPSHOT                               [3/49]
[INFO] --------------------------------[ jar ]---------------------------------
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-surefire-plugin/2.22.2/maven-surefire-plugin-2.22.2.pom
(...)
Downloaded from central: https://repo1.maven.org/maven2/com/phloc/phloc-commons/4.4.13/phloc-commons-4.4.13.jar (1.8 MB at 1.8 MB/s)
[INFO] 
[INFO] --- maven-clean-plugin:3.0.0:clean (default-clean) @ asqatasun-api ---
[INFO] 
[INFO] --- build-helper-maven-plugin:3.0.0:timestamp-property (timestamp-property) @ asqatasun-api ---
[INFO] 
[INFO] --- buildnumber-maven-plugin:1.4:create (default) @ asqatasun-api ---
[INFO] ShortRevision tag detected. The value is '7'.
[INFO] Executing: /bin/sh -c cd '/Users/a/github/Asqatasun/engine/asqatasun-api' && 'git' 'rev-parse' '--verify' '--short=7' 'HEAD'
[INFO] Working directory: /Users/a/github/Asqatasun/engine/asqatasun-api
[INFO] Storing buildNumber: 2795adf at timestamp: 1585118777309
[INFO] Storing buildScmBranch: develop
[INFO] 
[INFO] --- jacoco-maven-plugin:0.8.5:prepare-agent (default) @ asqatasun-api ---
[INFO] argLine set to -javaagent:/Users/a/.m2/repository/org/jacoco/org.jacoco.agent/0.8.5/org.jacoco.agent-0.8.5-runtime.jar=destfile=/Users/a/github/Asqatasun/engine/asqatasun-api/target/jacoco.exec
[INFO] 
[INFO] --- markdown-page-generator-plugin:2.1.0:generate (default) @ asqatasun-api ---
[INFO] Pre-processing markdown files from input directory: /Users/a/github/Asqatasun/engine/asqatasun-api
[INFO] 
[INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ asqatasun-api ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] skip non existing resourceDirectory /Users/a/github/Asqatasun/engine/asqatasun-api/src/main/resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.8.1:compile (default-compile) @ asqatasun-api ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 234 source files to /Users/a/github/Asqatasun/engine/asqatasun-api/target/classes
[INFO] /Users/a/github/Asqatasun/engine/asqatasun-api/src/main/java/org/asqatasun/util/http/Downloader.java: /Users/a/github/Asqatasun/engine/asqatasun-api/src/main/java/org/asqatasun/util/http/Downloader.java uses or overrides a deprecated API.
[INFO] /Users/a/github/Asqatasun/engine/asqatasun-api/src/main/java/org/asqatasun/util/http/Downloader.java: Recompile with -Xlint:deprecation for details.
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /Users/a/github/Asqatasun/engine/asqatasun-api/src/main/java/org/asqatasun/service/ContentAdapterService.java:[26,33] package javax.xml.bind.annotation does not exist
[ERROR] /Users/a/github/Asqatasun/engine/asqatasun-api/src/main/java/org/asqatasun/service/ContentAdapterService.java:[39,2] cannot find symbol
  symbol: class XmlTransient
[ERROR] /Users/a/github/Asqatasun/engine/asqatasun-api/src/main/java/org/asqatasun/service/CrawlerService.java:[25,33] package javax.xml.bind.annotation does not exist
[ERROR] /Users/a/github/Asqatasun/engine/asqatasun-api/src/main/java/org/asqatasun/service/CrawlerService.java:[37,2] cannot find symbol
  symbol: class XmlTransient
[ERROR] /Users/a/github/Asqatasun/engine/asqatasun-api/src/main/java/org/asqatasun/service/ContentLoaderService.java:[26,33] package javax.xml.bind.annotation does not exist
[ERROR] /Users/a/github/Asqatasun/engine/asqatasun-api/src/main/java/org/asqatasun/service/ContentLoaderService.java:[35,2] cannot find symbol
  symbol: class XmlTransient
[ERROR] /Users/a/github/Asqatasun/engine/asqatasun-api/src/main/java/org/asqatasun/service/AuditService.java:[27,33] package javax.xml.bind.annotation does not exist
[ERROR] /Users/a/github/Asqatasun/engine/asqatasun-api/src/main/java/org/asqatasun/service/AuditService.java:[35,2] cannot find symbol
  symbol: class XmlTransient
[ERROR] /Users/a/github/Asqatasun/engine/asqatasun-api/src/main/java/org/asqatasun/consolidator/ConsolidatorService.java:[27,33] package javax.xml.bind.annotation does not exist
[ERROR] /Users/a/github/Asqatasun/engine/asqatasun-api/src/main/java/org/asqatasun/consolidator/ConsolidatorService.java:[34,2] cannot find symbol
  symbol: class XmlTransient
[ERROR] /Users/a/github/Asqatasun/engine/asqatasun-api/src/main/java/org/asqatasun/service/ProcessorService.java:[25,33] package javax.xml.bind.annotation does not exist
[ERROR] /Users/a/github/Asqatasun/engine/asqatasun-api/src/main/java/org/asqatasun/service/ProcessorService.java:[34,2] cannot find symbol
  symbol: class XmlTransient
[ERROR] /Users/a/github/Asqatasun/engine/asqatasun-api/src/main/java/org/asqatasun/analyser/AnalyserService.java:[27,33] package javax.xml.bind.annotation does not exist
[ERROR] /Users/a/github/Asqatasun/engine/asqatasun-api/src/main/java/org/asqatasun/analyser/AnalyserService.java:[33,2] cannot find symbol
  symbol: class XmlTransient
[ERROR] /Users/a/github/Asqatasun/engine/asqatasun-api/src/main/java/org/asqatasun/service/ScenarioLoaderService.java:[25,33] package javax.xml.bind.annotation does not exist
[ERROR] /Users/a/github/Asqatasun/engine/asqatasun-api/src/main/java/org/asqatasun/service/ScenarioLoaderService.java:[34,2] cannot find symbol
  symbol: class XmlTransient
[INFO] 16 errors 
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] asqatasun 5.0-SNAPSHOT ............................. SUCCESS [ 17.667 s]
[INFO] engine 5.0-SNAPSHOT ................................ SUCCESS [  0.030 s]
[INFO] asqatasun-api 5.0-SNAPSHOT ......................... FAILURE [  6.226 s]
[INFO] rule-implementation-loader 5.0-SNAPSHOT ............ SKIPPED
[INFO] nomenclatureloader 5.0-SNAPSHOT .................... SKIPPED
[INFO] asqatasun-model 5.0-SNAPSHOT ....................... SKIPPED
[INFO] contentloader 5.0-SNAPSHOT ......................... SKIPPED
[INFO] contentadapter 5.0-SNAPSHOT ........................ SKIPPED
[INFO] commons-processing 5.0-SNAPSHOT .................... SKIPPED
[INFO] processor 5.0-SNAPSHOT ............................. SKIPPED
[INFO] consolidator 5.0-SNAPSHOT .......................... SKIPPED
[INFO] asqatasun-services 5.0-SNAPSHOT .................... SKIPPED
[INFO] analyser 5.0-SNAPSHOT .............................. SKIPPED
[INFO] logging 5.0-SNAPSHOT ............................... SKIPPED
[INFO] scenarioloader 5.0-SNAPSHOT ........................ SKIPPED
[INFO] asqatasun-engine 5.0-SNAPSHOT ...................... SKIPPED
[INFO] asqatasun-resources 5.0-SNAPSHOT ................... SKIPPED
[INFO] asqatasun-persistence 5.0-SNAPSHOT ................. SKIPPED
[INFO] testing-tools 5.0-SNAPSHOT ......................... SKIPPED
[INFO] rules-testing-tools 5.0-SNAPSHOT ................... SKIPPED
[INFO] asqatasun-maven-kbcsv-plugin 5.0-SNAPSHOT .......... SKIPPED
[INFO] asqatasun-maven-testgenerator-plugin 5.0-SNAPSHOT .. SKIPPED
[INFO] tgol-testing-tools 5.0-SNAPSHOT .................... SKIPPED
[INFO] rules-doc-utils 5.0-SNAPSHOT ....................... SKIPPED
[INFO] rules 5.0-SNAPSHOT ................................. SKIPPED
[INFO] rules-commons 5.0-SNAPSHOT ......................... SKIPPED
[INFO] rules-creation-demo 5.0-SNAPSHOT ................... SKIPPED
[INFO] rules-resources 5.0-SNAPSHOT ....................... SKIPPED
[INFO] rules-rgaa3.0 5.0-SNAPSHOT ......................... SKIPPED
[INFO] rules-rgaa3.2016 5.0-SNAPSHOT ...................... SKIPPED
[INFO] rules-rgaa3.2017 5.0-SNAPSHOT ...................... SKIPPED
[INFO] rules-accessiweb2.2 5.0-SNAPSHOT ................... SKIPPED
[INFO] rules-seo1.0 5.0-SNAPSHOT .......................... SKIPPED
[INFO] web-app 5.0-SNAPSHOT ............................... SKIPPED
[INFO] email-sender 5.0-SNAPSHOT .......................... SKIPPED
[INFO] security 5.0-SNAPSHOT .............................. SKIPPED
[INFO] tgol-resources 5.0-SNAPSHOT ........................ SKIPPED
[INFO] tgol-api 5.0-SNAPSHOT .............................. SKIPPED
[INFO] tgol-si 5.0-SNAPSHOT ............................... SKIPPED
[INFO] tgol-data-presentation 5.0-SNAPSHOT ................ SKIPPED
[INFO] tgol-persistence 5.0-SNAPSHOT ...................... SKIPPED
[INFO] tgol-orchestrator 5.0-SNAPSHOT ..................... SKIPPED
[INFO] tgol-voter 5.0-SNAPSHOT ............................ SKIPPED
[INFO] tgol-report 5.0-SNAPSHOT ........................... SKIPPED
[INFO] tgol-security 5.0-SNAPSHOT ......................... SKIPPED
[INFO] asqatasun-web-app 5.0-SNAPSHOT ..................... SKIPPED
[INFO] runner 5.0-SNAPSHOT ................................ SKIPPED
[INFO] asqatasun-runner 5.0-SNAPSHOT ...................... SKIPPED
[INFO] asqatasun-documentation 0.5.2-dev .................. SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  24.259 s
[INFO] Finished at: 2020-03-25T02:46:18-04:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.1:compile (default-compile) on project asqatasun-api: Compilation failure: Compilation failure: 
[ERROR] /Users/a/github/Asqatasun/engine/asqatasun-api/src/main/java/org/asqatasun/service/ContentAdapterService.java:[26,33] package javax.xml.bind.annotation does not exist
[ERROR] /Users/a/github/Asqatasun/engine/asqatasun-api/src/main/java/org/asqatasun/service/ContentAdapterService.java:[39,2] cannot find symbol
[ERROR]   symbol: class XmlTransient
[ERROR] /Users/a/github/Asqatasun/engine/asqatasun-api/src/main/java/org/asqatasun/service/CrawlerService.java:[25,33] package javax.xml.bind.annotation does not exist
[ERROR] /Users/a/github/Asqatasun/engine/asqatasun-api/src/main/java/org/asqatasun/service/CrawlerService.java:[37,2] cannot find symbol
[ERROR]   symbol: class XmlTransient
[ERROR] /Users/a/github/Asqatasun/engine/asqatasun-api/src/main/java/org/asqatasun/service/ContentLoaderService.java:[26,33] package javax.xml.bind.annotation does not exist
[ERROR] /Users/a/github/Asqatasun/engine/asqatasun-api/src/main/java/org/asqatasun/service/ContentLoaderService.java:[35,2] cannot find symbol
[ERROR]   symbol: class XmlTransient
[ERROR] /Users/a/github/Asqatasun/engine/asqatasun-api/src/main/java/org/asqatasun/service/AuditService.java:[27,33] package javax.xml.bind.annotation does not exist
[ERROR] /Users/a/github/Asqatasun/engine/asqatasun-api/src/main/java/org/asqatasun/service/AuditService.java:[35,2] cannot find symbol
[ERROR]   symbol: class XmlTransient
[ERROR] /Users/a/github/Asqatasun/engine/asqatasun-api/src/main/java/org/asqatasun/consolidator/ConsolidatorService.java:[27,33] package javax.xml.bind.annotation does not exist
[ERROR] /Users/a/github/Asqatasun/engine/asqatasun-api/src/main/java/org/asqatasun/consolidator/ConsolidatorService.java:[34,2] cannot find symbol
[ERROR]   symbol: class XmlTransient
[ERROR] /Users/a/github/Asqatasun/engine/asqatasun-api/src/main/java/org/asqatasun/service/ProcessorService.java:[25,33] package javax.xml.bind.annotation does not exist
[ERROR] /Users/a/github/Asqatasun/engine/asqatasun-api/src/main/java/org/asqatasun/service/ProcessorService.java:[34,2] cannot find symbol
[ERROR]   symbol: class XmlTransient
[ERROR] /Users/a/github/Asqatasun/engine/asqatasun-api/src/main/java/org/asqatasun/analyser/AnalyserService.java:[27,33] package javax.xml.bind.annotation does not exist
[ERROR] /Users/a/github/Asqatasun/engine/asqatasun-api/src/main/java/org/asqatasun/analyser/AnalyserService.java:[33,2] cannot find symbol
[ERROR]   symbol: class XmlTransient
[ERROR] /Users/a/github/Asqatasun/engine/asqatasun-api/src/main/java/org/asqatasun/service/ScenarioLoaderService.java:[25,33] package javax.xml.bind.annotation does not exist
[ERROR] /Users/a/github/Asqatasun/engine/asqatasun-api/src/main/java/org/asqatasun/service/ScenarioLoaderService.java:[34,2] cannot find symbol
[ERROR]   symbol: class XmlTransient
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :asqatasun-api
```

It may be a few days to a week before I have time to work on this again (I'm running `mvn package` in HTMLUnit right now so I'll see if I can get that to finish successfully in case that's the problem) so forgive me if I don't reply right away.

Thanks for your help.

**Comment by dzc34** [comment]
@jkowalczyk and @adaminfinitum:
same as issue #289 "Build fail: "javax.xml.bind.annotation does not exist" (openjdk 11, maven 3.6)"

> ``` 
> java.xml.bind (JAXB)
>     Java 8 - OK
>     Java 9 - DEPRECATED
>     Java 10 - DEPRECATED
>     Java 11 - REMOVED
> ```
> source: https://stackoverflow.com/questions/52502189/java-11-package-javax-xml-bind-does-not-exist/52502208#52502208
> ```
> <dependency>
>   <groupId>javax.xml.bind</groupId>
>   <artifactId>jaxb-api</artifactId>
>   <version>2.3.1</version>
> </dependency>
> 
> ```
> source: https://stackoverflow.com/questions/43574426/how-to-resolve-java-lang-noclassdeffounderror-javax-xml-bind-jaxbexception-in-j/52872160#52872160

**Comment by dzc34** [comment]
@adaminfinitum, if you want, you can create a new issue for this build problem on the develop branch.
> same as issue #289 "Build fail: "javax.xml.bind.annotation does not exist" (openjdk 11, maven 3.6)"

**Comment by jkowalczyk** [comment]
Why does the docker image run with openjdk 11?

**Comment by dzc34** [comment]
maven build is not include in docker container.

try this :
```bash
cd ~ && mv -v m2 m2_backup
cd Asqatasun && mv clean install
```

**Comment by adaminfinitum** [comment]
@dzc34 Is that really supposed to be `mv clean install`?

I'm assuming you mean `mvn clean install` (after I have moved or removed `~/.m2`) but that's the same command I ran the first time so should I fetch and merge first?

My system Java is OpenJDK-13 but I do have OpenJDK-8 installed via a package manager, do I need to tell Maven to use OpenJDK-8? And if so, how would I do that?

I have a built image of `asqatasun/asqatasun  v4.1.0-rc.4` on my system from running `docker pull ...` but not containers built from it, do I need to delete that?
